src/view/utils/fetchFromPlatform.js

```diff
@@ -63,17 +67,25 @@ export default async ({ orgId, imsAccess, path, params, headers, signal }) => {
   const host = getHost({ imsAccess });
   const baseRequestHeaders = getBaseRequestHeaders({ orgId, imsAccess });
   const queryString = params ? `?${params.toString()}` : "";
+  let response;
 
-  const response = await fetch(`${host}${path}${queryString}`, {
-    headers: {
-      ...baseRequestHeaders,
-      ...headers
-    },
-    signal
-  });
+  try {
+    response = await fetch(`${host}${path}${queryString}`, {
+      headers: {
+        ...baseRequestHeaders,
+        ...headers
+      },
+      signal
+    });
+  } catch (e) {
+    if (e.name === "AbortError") {
+      throw e;
+    }
+    throw new UserReportableError(ERROR_UNABLE_TO_CONNECT_TO_SERVER);
```

Comment:
I added this in case the user didn't have an internet connection or the CORS request failed or something like that.